profile/Client/profileClient.go

```diff
@@ -411,9 +416,16 @@ func generateRowsFromData(data []pb.Log, Operation string) []table.Row {
 				(entry.ContainerName == o1.Container) ||
 				(len(o1.Namespace) == 0 && len(o1.Pod) == 0 && len(o1.Container) == 0) {
 				var p Profile
+				var logType string
```

Comment:
Maybe we can add in this condition for namespace and container_name? Since they don't exist in Host logs, we can just put them as "--" instead of empty space